### PR TITLE
Add Jakartanet nodes

### PR DESCRIPTION
**Pull request by smondet**

### CODE REVIEW

**Comment by smondet** [comment]
Basic test with `KT1A83HF5DkY7hQriRZwWesMbVmjftEGGUy7` on Jakartanet (can't find any explorer)

File: src/client/query_nodes.ml [hunk 1]
```diff
@@ -305,6 +305,8 @@ let default_nodes : Node.t list =
   List.rev
     [ Node.create "Ithacanet-Teztnets" ~network:`Ithacanet
         "https://rpc.ithacanet.teztnets.xyz/" ~info_url:teztnets
+    ; Node.create "Jakartanet-Teztnets" ~network:`Jakartanet
```
Comment (marklnichols) [inline src/client/query_nodes.ml:308]: For whatever reason, there is no corresponding Granadanet to remove here, right?

**Review by marklnichols — APPROVED** [review]

File: src/client/query_nodes.ml (hunk 1 above)
Comment (smondet) [reply src/client/query_nodes.ml:308]: indeed teztnets started at Ithacanet

**Comment by smondet** [comment]
Deployed to staging: https://tzcomet.io/staging/#/explorer%3Fexplorer-input%3DKT1A83HF5DkY7hQriRZwWesMbVmjftEGGUy7

**Comment by smondet** [comment]
And to prod: <https://tzcomet.io/#/explorer%3Fexplorer-input%3DKT1A83HF5DkY7hQriRZwWesMbVmjftEGGUy7>

**Pull request merged**